tests/test_argument_parsing.py

```diff
@@ -617,9 +617,32 @@ def test_disk_image_path(target, expected_name):
 
 
 @pytest.mark.parametrize("target,config_options,expected_name,extra_kernels", [
+    # RISCV kernconf tests
     pytest.param("cheribsd-riscv64-purecap", [], "CHERI-QEMU", []),
+    pytest.param("cheribsd-riscv64-purecap", ["--cheribsd/build-fpga-kernels"], "CHERI-QEMU", []),
+    pytest.param("cheribsd-riscv64-purecap", ["--cheribsd/build-alternate-abi-kernels"],
+                 "CHERI-QEMU", ["CHERI-PURECAP-QEMU"]),
+    pytest.param("cheribsd-riscv64-purecap",
+                 ["--cheribsd/build-alternate-abi-kernels",
+                  "--cheribsd/default-kernel-abi", "purecap"],
+                 "CHERI-PURECAP-QEMU", ["CHERI-QEMU"]),
+    pytest.param("cheribsd-riscv64-purecap", ["--cheribsd/build-fett-kernels"],
+                 "CHERI-QEMU-FETT", ["CHERI-QEMU"]),
+    pytest.param("cheribsd-riscv64-purecap",
+                 ["--cheribsd/build-fett-kernels",
+                  "--cheribsd/build-alternate-abi-kernels"],
+                 "CHERI-QEMU-FETT", ["CHERI-QEMU", "CHERI-PURECAP-QEMU"]),
```

Comment:
Maybe also add one RISC-V case to show that benchmark builds `CHERI-QEMU-NODEBUG`?

Reply:
Turns out this was missing in the generator. Now fixed.